### PR TITLE
Renderer fixes

### PR DESCRIPTION
**Pull request by ghost**
- Done requests listed in #3.
- Fixed a MP crash related to non existing global NPCs.
- Fixed a miracle blight visual issue.

### CODE REVIEW

**Comment by steviegt6** [comment]
Didn't mean to discard changes to CalamityGlobalNPC, oops; should be fixed.

File: Effects/MiracleBlight.xnb
Comment (steviegt6) [inline Effects/MiracleBlight.xnb:1]: Should we leave the compiled binary here?

File: Graphics/Renderers/BaseDrawer.cs [hunk 1]
```diff
@@ -30,27 +31,18 @@ public ManagedRenderTarget MainTarget
...
-        /// <summary>
-        /// Called on mod load.
-        /// </summary>
-        public virtual void OnLoad()
+        protected sealed override void Register()
         {
+            ModTypeLookup<BaseRenderer>.Register(this);
 
+            if (!RendererManager.Renderers.Contains(this))
+                RendererManager.Renderers.Add(this);
```
Comment (steviegt6) [inline Graphics/Renderers/BaseDrawer.cs]: Redundant, should error instead.

**Review by steviegt6 — DISMISSED** [review]

File: Effects/MiracleBlight.xnb
Comment (ghost) [reply Effects/MiracleBlight.xnb:1]: Unsure what you mean? It's been updated due to the .fx being edited.

Comment (Ozzatron) [reply Effects/MiracleBlight.xnb:1]: Every Calamity shader has its xnb file in the Effects folder. This is no different

**Review by Ozzatron — APPROVED** [review]
Current state of the PR looks good.

File: Effects/MiracleBlight.xnb
Comment (steviegt6) [reply Effects/MiracleBlight.xnb:1]: My mistake, at a glance it looked as though the file was *deleted* but it was just a reduction in file size. That's why I asked.

**Review by steviegt6 — APPROVED** [review]

**Pull request merged**